### DISCUSSION TITLE
Who is willing to act as the initial committer of Amoro?

### DISCUSSION THREAD

**Discussion by zhoujinsong** [apache/amoro#2104] [Announcements]
Hi Amoro community，

Since Amoro open-sourced in August 2022, with the help of all contributors and users, the Amoro community has been established. To ensure the long-term and healthy development of Amoro, the community is considering donating the project to the Apache Sofatware Foundation Incubator. I am preparing some preparatory work for entering the incubator and have completed the [Amoro Proposal draft](https://docs.google.com/document/d/1Lox_jrU_JVSKVuOVA_Y10SwHU2mIvXs7f4DEcbzzknE/edit?usp=sharing).

Here, I invite contributors who have made significant contributions to the project to act as the initial committer of the project, help the project complete the incubation process, and achieve long-term development. If you are willing to become an initial committer, please leave a comment below with your basic information. The project is expected to initiate incubation discussions in the Apache Incubator from October to November 2023. Before that, you can accumulate your contributions to the project and indicate your willingness below. I will review and update the proposal in a timely manner.

Here is a example of basic information we need:
```
I am willing to act as the initial committer of Amoro.
Name: Jinsong Zhou
E-mail: jinsong.zhou1990@gmail.com
Affiliation: Netease
```

**Comment by zhoujinsong** [comment]
I am willing to act as the initial committer of Amoro.
Name: Jinsong Zhou
E-mail: jinsong.zhou1990@gmail.com
Affiliation: Netease

**Comment by HuangFru** [comment]
I am willing to act as the initial committer of Amoro.
Name: Jianmin Huang
E-mail: [steves.hjm@gmail.com](mailto:steves.hjm@gmail.com)
Affiliation: Netease

**Comment by zhongqishang** [comment]
I am willing to act as the initial committer of Amoro.
Name: Qishang Zhong
E-mail: zhongqishang@gmail.com
Affiliation: Qichacha

**Comment by YesOrNo828** [comment]
I am willing to act as the initial committer of Amoro.
Name: Xianxun Ye
E-mail: yesorno828423@gmail.com
Affiliation: Netease

**Comment by hameizi** [comment]
I am willing to act as the initial committer of Amoro.
Name: Zeyu Wang
E-mail: hameizi369@gmail.com
Affiliation: Netease

**Comment by xxubai** [comment]
I am willing to act as the initial committer of Amoro.
Name: Xu Bai
E-mail: tocreationbai@gmail.com
Affiliation: Cisco Webex

**Comment by hellojinsilei** [comment]
I am willing to act as the initial committer of Amoro.
Name: Silei Jin
E-mail: [sileijin204@gmail.com](mailto:sileijin204@gmail.com)
Affiliation: Dtdream

**Comment by hzluting** [comment]
I am willing to act as the initial committer of Amoro.
Name: Ting Lu
E-mail: [dylzlt93299@gmail.com](mailto:dylzlt93299@gmail.com)
Affiliation: Netease

**Comment by shidayang** [comment]
I am willing to act as the initial committer of Amoro.
Name: Dayang Shi
E-mail: jocean.shi@gmail.com
Affiliation: Netease

**Comment by baiyangtx** [comment]
I am willing to act as the initial committer of Amoro.
Name: Yongxiang Zhang
E-mail: baiyangtx@gmail.com
Affiliation: Netease

**Comment by wangtaohz** [comment]
I am willing to act as the initial committer of Amoro.
Name: Tao Wang
E-mail: [tao.wang.hz@gmail.com](mailto:tao.wang.hz@gmail.com)
Affiliation: Netease

**Comment by majin1102** [comment]
I am willing to act as the initial committer of Amoro.
Name: Nathan Ma
E-mail: [majin1102@gmail.com](majin1102@gmail.com)
Affiliation: Netease

**Comment by shendanfengg** [comment]
I am glad to know that Amoro is going to be donated to the Apache Software Foundation incubator. Good luck!

**Comment by huyuanfeng2018** [comment]
I am willing to act as the initial committer of Amoro.
Name: Yuanfeng Hu
E-mail: [heigebupahei@gmail.com](mailto:heigebupahei@gmail.com)
Affiliation: Huya

**Comment by czy006** [comment]
I am willing to act as the initial committer of Amoro.
Name: ZhengYu Chen
E-mail: [jam.gzczy@gmail.com](mailto:jam.gzczy@gmail.com)
Affiliation: HuoLaLa

**Comment by tcodehuber** [comment]
I am willing to act as the initial committer of Amoro.
Name: Gang Huang
E-mail: [tcodehuber@gmail.com](mailto:tcodehuber@gmail.com)
Affiliation: Dmall
